great_expectations/data_context/data_context.py

```diff
@@ -941,6 +936,17 @@ def profiler_store(self) -> ProfilerStore:
                 f'Attempted to access the Profiler store named "{profiler_store_name}", which is not a configured store.'
             )
 
+    @staticmethod
+    def default_profilers_exist(directory_path: Optional[str]) -> bool:
+        if not directory_path:
+            return False
+
+        profiler_directory_path: str = os.path.join(
+            directory_path,
+            DataContextConfigDefaults.DEFAULT_PROFILER_STORE_BASE_DIRECTORY_RELATIVE_NAME.value,
+        )
+        return os.path.isdir(profiler_directory_path)
```

Comment:
This method was in the RBP toolkit but makes more sense in DataContext since it is directly asking the question of whether or not the context's profiler directory is present.

Reply:
I'd be tempted to put this on the RulesBasedProfiler as a static method along with the constant; there's no reason the component cannot own this IMHO

Reply:
Can this be made "private" with an _? I consider anything without an underscore to be part of the public API, and I want to keep that surface as small as possible. Unless we expect GE users to use it, it should be private